gui/gui/src/ts/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+
+import {Typography, AppBar, Toolbar} from "@material-ui/core";
+
+export const Navbar: React.FunctionComponent<{}> = () => {
+  return (
+    <AppBar position="static">
+      <Toolbar>
+        <Typography variant="h6">Mowgli</Typography>
```

Comment:
Nit: MOWGLI is an acronym (actually a backronym), but don't worry about it for this PR.